BoxContentSDK/BoxContentSDK/BOXContentSDKConstants.m

```diff
@@ -154,7 +154,8 @@
 NSString *const BOXRepresentationTemplateKeyAccessPath = @"{+asset_path}";
 
 // Representation Template Value
-NSString *const BOXRepresentationTemplateValueHLSManifiest = @"master.m3u8";
+NSString *const BOXRepresentationTemplateValueHLSManifest = @"master.m3u8";
+NSString *const BOXRepresentationTemplateValueMP4Manifest = @"";
```

Comment:
Probably not needed, This would be starting a precedence to define manifests files for assets without manifests.